### PR TITLE
Frt 5688

### PR DESCRIPTION
**Pull request by laniakea42**

### CODE REVIEW

File: packages/landing/public/fonts/slick.svg
Comment (bouvens) [inline packages/landing/public/fonts/slick.svg:1]: Where this file is used?

File: packages/landing/public/fonts/slick.ttf
Comment (bouvens) [inline packages/landing/public/fonts/slick.ttf:1]: and this

File: packages/landing/public/fonts/slick.woff
Comment (bouvens) [inline packages/landing/public/fonts/slick.woff:1]: and this

File: packages/landing/src/app/common/space.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+const Space = styled.div<{
+  height?: number
+  heightMobile?: number
+  width?: number
+  widthMobile?: number
+  xl?: boolean
+  md?: boolean
+  sm?: boolean
+  xs?: boolean
```
Comment (bouvens) [inline packages/landing/src/app/common/space.tsx:11]: Let's left only `height` and `heightMobile` properties that we actually use.

File: packages/landing/src/app/common/space.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    @media (max-width: 1023px) {
+      height: ${heightMobile !== undefined ? heightMobile : 40}px;
+      ${sm || xs ? 'height: 24px' : ''};
+      width: ${widthMobile !== undefined ? widthMobile : 0}px;
+    }
+  `,
+)
+
+export default Space
```
Comment (bouvens) [inline packages/landing/src/app/common/space.tsx:29]: Let's not use default exports for components for simplify consistent renaming and finding usages with IDE.

File: packages/landing/src/app/style.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
+@font-face {
+  font-family: "PlusJakartaSans";
+  font-display: swap;
+  src: url("/fonts/PlusJakartaSans-VariableFont_wght.ttf") format("truetype");
```
Comment (bouvens) [inline packages/landing/src/app/style.css]: Could you please use `next/font` as in the previous PR?

File: packages/landing/src/app/style.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+h6 {
+  font-weight: normal;
+  text-shadow: 0 0 0;
+  font-family:"PlusJakartaSans", sans-serif;
+
+}
+
+p,
+a {
+  /* color: var(--text-primary); */
```
Comment (bouvens) [inline packages/landing/src/app/style.css:36]: Do we need this?

File: packages/landing/src/app/page.tsx [hunk 5]
```diff
@@ -1,93 +1,146 @@
...
+import './style.css'
+import Space from './common/space'
+import { useState, useRef } from 'react'
+import { useClickOutside } from './use-click-outside'
+import cn from 'classnames'
 
 export default function Home() {
+  const [email, setEmail] = useState('')
+
+  const [isEmailFocused, setIsEmailFocesed] = useState(false)
```
Comment (bouvens) [inline packages/landing/src/app/page.tsx:38]:
```suggestion
  const [isEmailFocused, setIsEmailFocused] = useState(false)
```

File: packages/landing/src/app/page.tsx [hunk 6]
```diff
@@ -1,93 +1,146 @@
...
+  const [isEmailError, setIsEmailError] = useState(false)
+  const ref = useRef(null)
+
+  useClickOutside(ref, () => setIsEmailFocesed(false))
```
Comment (bouvens) [inline packages/landing/src/app/page.tsx:42]: Can't we simply use `onFocus` and `onBlur` props?

**Review by bouvens — CHANGES_REQUESTED** [review]

File: packages/landing/public/fonts/slick.svg
Comment (laniakea42) [reply packages/landing/public/fonts/slick.svg:1]: Nowhere, I've deleted it

File: packages/landing/public/fonts/slick.ttf
Comment (laniakea42) [reply packages/landing/public/fonts/slick.ttf:1]: Nowhere, I've deleted it

File: packages/landing/public/fonts/slick.woff
Comment (laniakea42) [reply packages/landing/public/fonts/slick.woff:1]: Nowhere, I've deleted it

File: packages/landing/src/app/common/space.tsx (hunk 1 above)
Comment (laniakea42) [reply packages/landing/src/app/common/space.tsx]: Ok, done it

File: packages/landing/src/app/common/space.tsx (hunk 2 above)
Comment (laniakea42) [reply packages/landing/src/app/common/space.tsx]: Ok, fixed

File: packages/landing/src/app/style.css (hunk 3 above)
Comment (laniakea42) [reply packages/landing/src/app/style.css]: Ok, fixed

File: packages/landing/src/app/page.tsx (hunk 5 above)
Comment (laniakea42) [reply packages/landing/src/app/page.tsx:37]: I've fixed this typo  thanks

File: packages/landing/public/fonts/slick.svg
Comment (bouvens) [reply packages/landing/public/fonts/slick.svg:1]: Thank you

**Review by bouvens — APPROVED** [review]

**Pull request merged**